great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -54,11 +54,13 @@ class ParameterBuilder(Builder, ABC):
 
     exclude_field_names: Set[str] = {
         "data_context",
+        "batch_list",
     }
```

Comment:
@cdkini This is necessary to be in compliance with `SerializableDictDot`, so we excluded non-serializable fields.

Reply:
This is fine - just playing devil's advocate, is there a better way to identify non-serializable fields? Perhaps iterate through the keys, pass it through some black box to determine serializability, and convert as applicable?

Reply:
@cdkini That (the idea about the serializability filter) sounds like a really good idea.  We could still leave the same interface that requires the fields to be specified at the class level; however, the fields would be computed, rather than stated as a constant.  Adding it to backlog.  Thanks!